docs/docs/Integrations/Apify/integrations-apify.md

```diff
@@ -5,59 +5,65 @@ slug: /integrations-apify
 
 ## Example flows
 
-### Extract website text content in Markdown format
+Here are some example flows that use the **Apify Actors** component.
+
+### Extract website text content in Markdown
 
 Use the [Website Content Crawler Actor](https://apify.com/apify/website-content-crawler) to extract text content in Markdown format from a website and process it in your flow.
+
 ![Apify Flow - Website Content Crawler](./apify_flow_wcc.png)
 
-### Analyze and process website content with an Agent
+### Process web content with an Agent
+
+Extract website content using the [Website Content Crawler Actor](https://apify.com/apify/website-content-crawler), and then process  it with an agent.
+
+The agent takes the extracted data and transforms it into summaries, insights, or structured responses to make the information more actionable.
 
-Building on the previous example, this flow not only extracts website content using the [Website Content Crawler Actor](https://apify.com/apify/website-content-crawler) but also processes and analyzes it with an agent. The agent takes the extracted data and transforms it into summaries, insights, or structured responses, making the information more actionable. Unlike simple extraction, this approach enables automated content understanding and contextual processing.
 ![Apify Agent Flow - Simple](./apify_agent_flow_simple.png)
 
-### Search and analyze social media profiles with an Agent
+### Analyze social media profiles with multiple actors
 
-Perform comprehensive social media research with multiple Apify Actors. Start with the [Google Search Results Scraper Actor](https://apify.com/apify/google-search-scraper) to find relevant social media profiles, then use the [TikTok Data Extractor Actor](https://apify.com/clockworks/free-tiktok-scraper) to gather data and videos. The agent streamlines the process by collecting links from Google and retrieving content from TikTok, enabling deeper analysis of a person, brand, or topic.
-![Apify Agent Flow](./apify_agent_flow.png)
+Perform comprehensive social media research with multiple Apify Actors.
 
-## Components
+Add the [Google Search Results Scraper Actor](https://apify.com/apify/google-search-scraper) to find relevant social media profiles, then add the [TikTok Data Extractor Actor](https://apify.com/clockworks/free-tiktok-scraper) to gather data and videos.
 
-### Apify Actors
+The agent collects the links from Google and content from TikTok and analyzes the data to provide insights about a person, brand, or topic.
+![Apify Agent Flow](./apify_agent_flow.png)
 
-This component allows you to run Apify Actors in your flow. It can be used manually by providing run input or integrated as a tool for an AI Agent. When used with an AI Agent, the agent can leverage the Apify Actors to perform various tasks.
+## Parameters
 
```

Comment:
Per the Google style guide, you should not have "empty" headings. In other words, you need some text between this heading and the next heading.